api/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -17,11 +21,41 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Configuration for the OpenTelemetry tracer.
 //  [#extension: envoy.tracers.opentelemetry]
 message OpenTelemetryConfig {
+  // Configuration for sending traces to an upstream cluster over HTTP. Note that
+  // this will only be used if the grpc_service is not set.
+  // [#next-free-field: 6]
+  message HttpConfig {
+    // The upstream cluster that will receive OTLP traces over HTTP.
+    string cluster_name = 1;
+
+    // The path to the trace ingest endpoint. The path is appended with the host name configured in the cluster.
+    // Optional: If omitted, "/v1/traces" will be used as default.
+    string traces_path = 2;
+
+    // Sets the maximum duration in milliseconds that a response can take to arrive upon request.
+    google.protobuf.Duration timeout = 3 [(validate.rules).duration = {
+      required: true
+      gte {}
+    }];
+
+    // Custom headers (key-value) to be added to the OTLP HTTP request.
+    repeated core.v3.HeaderValue headers = 4;
```

Comment:
I think I look at it, but I was not sure how would the configuration looks like with it. 

If I change here to use `HeaderMap` instead of `HeaderValue`,  won't I end up with a config like this?

```
headers: # from opentelemetry.proto
  headers: # from HeaderMap
    - key: "Authorization"
      value: "My token"
```

Reply:
Since I changed now to have a `HttpService`,  the headers are now using the type `HeaderValueOption` similar to how it is on [main/api/envoy/config/route/v3/route_components.proto](https://github.com/envoyproxy/envoy/blob/main/api/envoy/config/route/v3/route_components.proto?rgh-link-date=2023-09-12T14%3A57%3A01Z)